infra/gp-cluster-logging-instance/values.yaml

```diff
@@ -2,40 +2,63 @@ infranodes:
   enabled: true
 clusterlogging:
   managementState: "Managed"
-elasticsearch:
-  retentionPolicy: 
-    application:
-      maxAge: 30d
-    infra:
-      maxAge: 30d
-    audit:
-      maxAge: 2d
-  resources:
-    limits:
-      memory: "16Gi"
-      cpu: "2"
-    requests:
-      cpu: "1"
-      memory: "16Gi"
-  nodeCount: 3       
-  storage:
+lokistack:
+  minio:
+    enabled: false
+  configuration:
+    runOnInfra: true
```

Comment:
Das ist Redundant mit infranodes.enabled.
Bitte eine Variante wählen. Auch in den Files wo es verwendet wird.